README.md

```diff
@@ -190,7 +190,9 @@ Caching
 
 Caching is available at the connector level if the connector implements `CacheToggle`. Connectors typically extend `CachingConnector` which implements [PSR-6][PSR-6]-compatible caching. Porter ships with just one cache implementation, `MemoryCache`, which stores data in memory but this can be substituted for any PSR-6 cache if the connector permits it.
 
-When available, the connector caches raw responses for each unique cache key. The cache key is comprised of the source and options parameters passed to `Connector::fetch`. Options are sorted before the cache key is created so the order of options are insignificant.
+When available, the connector caches raw responses for each unique [cache key](#cache-key).
```

Comment:
This should be expanded to explain that the cache key is generated by an implementation-defined strategy and name the default strategy.